popde/density_estimate.py

```diff
@@ -1,3 +1,128 @@
+class SimpleGaussianKernelDensityEstimation:
```

Comment:
I suggest just SimpleKDE, since the kernel is allowed to be something that is not 'gaussian', which would be confusing if 'Gaussian' is in the name

Reply:
renamed